app/controllers/support/courses_controller.rb

```diff
@@ -25,8 +25,12 @@ def update
 
   private
 
+    def recruitment_cycle
```

Comment:
Could we move this and the one below into `SupportController` as they both inherit from it

Reply:
Good shout. Done [here](https://github.com/DFE-Digital/publish-teacher-training/pull/2891/commits/2fbc00ca5d30ea288ce06c637bdfbea85cea0573)